bootstrap.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/sh
+
+#
+# Bootstrap 0.1.0 for reMarkable (rM2 untested)
+# wget -qO- '<URL_to_bootstrap.sh>' | sh -s [packages]...
+#
+
+bootstrap_wget() {
+  if [ ! -d /home/root/.wget_bin ]
```

Comment:
is there a common dir, like ~/.config/ or ~/.cache that can be used instead of putting .wget_bin in the toplevel namespace?

Reply:
Changed to ~/.cache/wget_bin in commit [b712a400](https://github.com/toltec-dev/toltec/pull/38/commits/b712a400c5ffe258e9cc5070adf0c5556af6c323) and [remarkable_entware/52e7c156](https://github.com/LinusCDE/remarkable_entware/commit/52e7c1566d83149894bb4a40b17432bd1c185038).